Client/Frontend/Browser/LoginsHelper.swift

```diff
@@ -228,7 +228,7 @@ class LoginsHelper: TabHelper {
             }
 
             let json = JSON(jsonObj)
-            let src = "window.__firefox__.logins.inject(\(json.rawString()))"
+            let src = "window.__firefox__.logins.inject(\(json.rawString()!))"
```

Comment:
Duh I mean we can assert it's not nil because it really should never be unless the app has run out of memory.